src/isomorphic/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -82,6 +82,15 @@ describe('ReactPropTypes', function() {
       );
     });
 
+    it('should warn for invalid functions', function() {
```

Comment:
I only see tests for `string`. I added `function` since I was relying on it in my tests. Is the thought to only test one example of `createPrimitiveTypeChecker`? If so I could remove mine.